src/crypto-api/index.ts

```diff
@@ -995,15 +1000,77 @@ export interface CrossSigningStatus {
 /**
  * Crypto callbacks provided by the application
  */
-export interface CryptoCallbacks extends SecretStorageCallbacks {
+export interface CryptoCallbacks {
```

Comment:
You intentionally removed the `extends SecretStorageCallbacks`? If so, maybe this should be described in the PR description, if it's significant.

Reply:
Ah, I see you have a comment that refers to this tangentially, and also that this is a draft PR, so I may have jumped the gun.